src/main/java/com/example/user_management/converter/UserConverter.java

```diff
@@ -0,0 +1,29 @@
+package com.example.user_management.converter;
+
+import com.example.user_management.entity.User;
+import com.example.user_management.model.request.CreateUserRequestModel;
+import org.springframework.stereotype.Component;
+
+import static org.springframework.util.Assert.hasLength;
+import static org.springframework.util.Assert.notNull;
+
+@Component
+public class UserConverter {
+
+    private UserConverter() {
+    }
```

Comment:
If you do use lombok why not to annotation with appropriate annotation and avoid constructor declaration?

Reply:
Done